Jenkinsfile

```diff
@@ -64,7 +64,7 @@ pipeline {
       }
       steps {
         script {
-          openshift.withCluster() {
+          openshift.withCluster("${env.DEV_NAMESPACE}") {
```

Comment:
Did you mean to reference the DEV_NAMESPACE in `openshift.WithCluster(...)`

Reply:
Good catch. Updated